integration-tests/benchmark_test.go

```diff
@@ -72,7 +77,12 @@ func (b *BenchmarkTestSuiteBase) StartPerfContainer(name string, image string, a
 }
 
 func (b *BenchmarkTestSuiteBase) RunInitContainer() {
-	init_image := qaImage("quay.io/rhacs-eng/collector-performance", "init")
+	image := "quay.io/rhacs-eng/collector-performance"
+
+	if runtime.GOARCH != "amd64" {
```

Comment:
same here .. see above.

Reply:
arch specifics have already been removed from here. Cant see this in the "Files Changed" tab.